ui/app/controllers/vault/cluster/access/mfa/methods.js

```diff
@@ -1,9 +1,7 @@
 import Controller from '@ember/controller';
 
 export default class MfaMethodsListController extends Controller {
-  queryParams = {
-    page: 'page',
-  };
```

Comment:
I believe that this is the format when defining query params on the route not the controller so I'm not sure if this ever worked but it was throwing an error that `queryParams` was not iterable.